packages/labs/react/README.md

```diff
@@ -127,6 +127,12 @@ Controller timings are implemented as follows:
 | requestUpdate    | useState setter                     |
 | updateComplete   | useLayoutEffect                     |
 
+### Limitations
+
+Because the controller's lifecycle is run via a `useLayoutEffect` callback, the initial value a controller produces for the React component may
+be incorrect. This is the case only if the controller sets up its initial value via `hostConnected` or `hostUpdate`. This limitation can be worked
```

Comment:
I wouldn't phrase this as "incorrect" here, but say that in React `hostConnected` is run after `hostUpdate` and to be careful about dependencies between the two. Ideally values produced by the controller are computed on demand so the order doesn't matter.

Reply:
Removed as this isn't the case with the current gambit.